### ISSUE TITLE
Check author's original implementation

### ISSUE THREAD

**Issue by sotetsuk** [sotetsuk/neural-language-correction#60]
Theano version: 

- https://bitbucket.org/zxie/nlc_models

Tensorflow reimplemented version: 
- https://github.com/sdlg/nlc

**Comment by sotetsuk** [comment]
List of differences: 

- [ ] Dropout
- [ ] Decoder's hidden states: hidden states are concatenated with context 💩 
  - https://github.com/sdlg/nlc/blob/master/nlc_model.py#L68
- [ ] Backpropagate through context to encoder (?)

**Comment by Doreenruirui** [comment]
Hi Sotetsuk,

Have you compared your result with the original implementation? How can I get the dataset you used here? Thank you very much!

Best,
Rui

**Comment by sotetsuk** [comment]
Hi, 

I'm sorry. The implementation of this repository is WIP, that is, not completed yet.
And there is no plan to restart the implementation.
I strongly recommend you to use the author's original TensorFlow implementation.

https://github.com/stanfordmlgroup/nlc

**Comment by Doreenruirui** [comment]
Hi sotetsuk,

Thank you very much for your reply. I will check the tensorflow version.

Best,
Rui
